src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import cn from 'classnames';
+import { FilterBy } from '../../types/FilterBy';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  filterBy: FilterBy,
+  setFilterBy: (param: FilterBy) => void
+  todos: Todo[]
+  removeCompletedTodos: () => void
+}
+
+export const Filter: React.FC<Props> = ({
+  filterBy,
+  setFilterBy,
+  todos,
+  removeCompletedTodos,
+}) => {
+  const leftTodos = todos.filter(todo => !todo.completed);
+
+  const enableButton = todos.some(todo => todo.completed);
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${leftTodos.length} items left`}
+      </span>
+
+      <nav className="filter">
+        <a
+          href="#/"
+          className={cn(
+            'filter__link',
+            { selected: filterBy === 'all' },
```

Comment:
No need to write 'all' by yourself you have created already `FilterBy.all` for this, use it.